app/src/main/java/com/weatherxm/util/ChartsAxisFormatters.kt

```diff
@@ -20,13 +19,11 @@ class CustomYAxisFormatter(
 ) : ValueFormatter() {
     @Suppress("MagicNumber")
     override fun getAxisLabel(value: Float, axis: AxisBase?): String {
-        val label = if (decimals > 0 && value < 10000) {
-            "${roundToDecimals(value, decimals)}"
-        } else if (value < 10000) {
-            "${roundToInt(value)}"
+        val label = if (value < 10000) {
+            formatNumber(value, decimals)
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/WeatherXM/wxm-android/pull/342#issuecomment-2388448877>#04ac52</a><span> - 10/02/2024, 12:12 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The change from roundToDecimals to formatNumber may alter the precision and formatting of the axis labels, potentially affecting the chart's readability and accuracy.
</div>
<div id="fix">
<b>Fix:</b> Add the 'enforceDecimals = true' parameter to the formatNumber function call to ensure that the specified number of decimal places is always used, maintaining the same precision as the previous roundToDecimals function.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -23,2 +23,2 @@
 -            formatNumber(value, decimals)
 +            formatNumber(value, decimals, enforceDecimals = true)
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged